.github/workflows/build-light-server-docker.yaml

```diff
@@ -0,0 +1,68 @@
+name: Build Light Docker
+
+on: workflow_dispatch
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  build-light-docker:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: "recursive"
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.9
+
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
```

Comment:
you can use uv  and change L30 & L31 with:

python -m pip install uv
python -m uv pip install .[all] -v
python -m uv pip install pyinstaller